xds/src/main/java/io/grpc/xds/GcpAuthenticationFilter.java

```diff
@@ -72,8 +72,8 @@ public ConfigOrError<? extends FilterConfig> parseFilterConfig(Message rawProtoM
 
     long cacheSize = 10;
     // Validate cache_config
-    TokenCacheConfig cacheConfig = gcpAuthnProto.getCacheConfig();
-    if (cacheConfig != null) {
+    if (gcpAuthnProto.hasCacheConfig()) {
+      TokenCacheConfig cacheConfig = gcpAuthnProto.getCacheConfig();
```

Comment:
nit: I will unwrap this to single line within cacheSize when I revisit this file for Audience parsing.